arch/common/gen_isr_tables.py

```diff
@@ -125,6 +125,45 @@ def parse_args():
 
     args = parser.parse_args()
 
+source_assembly_header = """
+#ifndef ARCH_IRQ_VECTOR_JUMP_CODE
+#define ARCH_IRQ_VECTOR_JUMP_CODE(v) "j " STRINGIFY(v)
+#endif
```

Comment:
This mysterious "j" instruction is ... rather confusing.

IMHO, `#error` would make more sense in this case since the arch port **must** provide the jump instruction for this to work properly (unless, of course, the arch just happens to have an mnemonic "j" that happens to be the jump instruction, which is quite unlikely and not something we can rely on ...).

Reply:
done